### ISSUE TITLE
don't save a submission until something is entered in the "Study" information page

### ISSUE THREAD

**Issue by mslarae13** [microbiomedata/issues#1004]
@pkalita-lbl is working on some update to the UI that will make "Study" the first page of information to complete when starting a submission. 

Currently as soon as you hit "+ Create Submission" it saves, assigns an ID, and you can go back into a totally empty submission.

Would it be possible to not save or assign an ID until someone enters something into this first page of the submission process?

**Comment by mslarae13** [comment]
low priority, definitely not until the external data squad is complete and PK has completed the UI change.

**Comment by pkalita-lbl** [comment]
I'm not suggesting we re-open this issue, but this is something we can think about as part of https://github.com/microbiomedata/nmdc-server/issues/1690
